content_handler/accessibility_bridge.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright 2017 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/content_handler/accessibility_bridge.h"
+
+#include <unordered_set>
+
+#include "apps/maxwell/services/context/context_writer.fidl.h"
+#include "apps/modular/lib/rapidjson/rapidjson.h"
```

Comment:
I'd recommend just using rapidjson directory.  These wrappers don't really add much (and aren't worth taking the dependency).

Reply:
Done!